src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -91,7 +89,14 @@ func DeduceBrokers(
 		}
 		// Otherwise, try to detect if a local container cluster is
 		// running, and use its brokers' addresses.
-		bs = ContainerBrokers()
+		bs, stopped := ContainerBrokers()
```

Comment:
the code looks good, but I think we are missing a test that tests for precedence rules of what configuration gets loaded first.

Reply:
Added tests.